platform/linux-generic/include/odp_buffer_internal.h

```diff
@@ -86,9 +79,17 @@ struct odp_buffer_hdr_t {
 	/* Event subtype. Should be ODP_EVENT_NO_SUBTYPE except packets. */
 	int8_t    event_subtype;
 
+#ifndef ODP_SCHEDULE_SCALABLE
```

Comment:
I suggest that we remove the #ifndef and the fields under this flag. This will help change the interface between packet I/O and the queues to use odp_event_t. This will further help in removing the conversions (between odp_buffer_hdr_t and odp_event_t) in scalable queue and scalable scheduler code.